### PR TITLE
Dockerize slashbase, update deprecated packages, update .gitignore file.

### PR DESCRIPTION
**Pull request by dozheiny**
I write Dockerfile for slash base; now you can run the project on Docker with the ` docker run  --name slashbase -p 22022:22022 -p 3000:3000 slashbase`  and building with `docker build  -t slashbase -f Dockerfile .` command.

Also, for limitations from Docker, I write supervisord.conf file for frontend and backend service running in the background.

Usually, when running the Docker service, I encountered an EOF problem because cmd was not receiving a value.
That's why I added a new env called cli so that only the web will run when CLI is false, and if CLI is true, CLI and web will run together.

In some functions, the slash base used deprecated packages for Golang version 1.16, so I replaced them with new packages.

Also, update the .gitingore file, adding JetBrains products environments.

And do some error handling.

### CODE REVIEW

File: cmd/cli.go [hunk 1]
```diff
@@ -114,7 +114,12 @@ func runQuery(queryCmd string) {
 func getQueryConfigs(dbConn *qemodels.DBConnection) *qemodels.QueryConfig {
 	createLog := func(query string) {
 		queryLog := models.NewQueryLog(dbConn.ID, query)
-		go dao.DBQueryLog.CreateDBQueryLog(queryLog)
+		go func() {
```
Comment (paraswaykole) [inline cmd/cli.go:117]: The code added is same as code removed. Except, the first code was just one line.
Doesn't make sense to write this.

**Comment by paraswaykole** [comment]
This is a huge PR. I will take some time to review this.

Thanks for the PR 🙏

**Comment by paraswaykole** [comment]
I may not be able to accept all of this, so I may request you to make changes.

File: cmd/cli.go (hunk 1 above)
Comment (dozheiny) [reply cmd/cli.go:117]: Yes, your right.

**Comment by dozheiny** [comment]
> I may not be able to accept all of this, so I may request you to make changes.

Sure, thanks for reviewing my PR.

**Comment by dozheiny** [comment]
I will give another PR to dockize the project.

**Pull request closed without merging**